pai-management/bootstrap/hadoop-service/hadoop-configuration/nodemanager-generate-script.sh

```diff
@@ -34,3 +34,11 @@ sed  -i "s/{HDFS_ADDRESS}/${HDFS_ADDRESS}/g" $HADOOP_CONF_DIR/core-site.xml
 
 sed  -i "s/{LOGSERVER_ADDRESS}/${LOGSERVER_ADDRESS}/g" $HADOOP_CONF_DIR/mapred-site.xml 
 
+# set memory and cpu resource for nodemanager
+mem_available=`cat /proc/meminfo | grep "MemTotal" | awk '{print $2}'`
+# memory size to nodemanager is floor(available_memory * 0.8)
+let mem_available=mem_available*8/10/1024/1024*1024
```

Comment:
You'd better unify the variable name's format with your data source. It will be better to use mem_total instead of mem_available.

Reply:
OK, changed.